src/js/components/RadioButtonGroup/RadioButtonGroup.js

```diff
@@ -17,6 +17,7 @@ const RadioButtonGroup = forwardRef(
   (
     {
       children,
+      defaultValue,
```

Comment:
I think we need to add `defaultValue` to doc.js.

Reply:
Should we add it to TextInput and TextArea too?

Reply:
I think we should, otherwise, how would users be aware of this optional setting?

Reply:
It's a standard DOM attribute. So, just like you can put an `id` on any component, we don't document it.